From a7a1eceee21ffb6691b86996838beea13f3d7b6f Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Wed, 4 Sep 2024 11:46:01 -0400
Subject: Use "WX_CPPFLAGS" instead of "WX_CXXFLAGS" in Makefile.am

This is needed to build mingw32 and to build using wxwidgets 3.3.x under mingw64.

---
 src/src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/src/Makefile.am b/src/src/Makefile.am
index 49d5dd98b..d59889bf5 100644
--- a/src/src/Makefile.am
+++ b/src/src/Makefile.am
@@ -75,7 +75,7 @@ codeblocks_LDADD += resources.$(OBJEXT) -lexchndl -lcomctl32
 codeblocks_DEPENDENCIES = resources.$(OBJEXT)
 codeblocks_SOURCES += associations.cpp
 resources.$(OBJEXT): resources/resources.rc resources/amd64_dpi_aware_on.manifest
-	$(WINDRES) $(WX_CXXFLAGS) -DcbDPI_AWARE_ON --include-dir $(top_srcdir)/src $< $@
+	$(WINDRES) $(WX_CPPFLAGS) -DcbDPI_AWARE_ON --include-dir $(top_srcdir)/src $< $@
 endif
 
 noinst_HEADERS = app.h \
-- 
2.45.2.windows.1

